### PR TITLE
Split SettingsMS2Deepscore and SettingsEvaluationModel

### PR DESCRIPTION
**Pull request by niekdejonge**
Continues in changes made in #206, I made a new PR to keep a clear overview. 

Here I split the SettingsMS2Deepscore and SettingsEvaluationModel. Let me know what you think.

### CODE REVIEW

File: ms2deepscore/train_new_model/spectrum_pair_selection.py [hunk 1]
```diff
@@ -150,13 +152,14 @@ def compute_fingerprint_dataframe(
         The settings that should be used for selecting the compound pairs wrapper. The settings should be specified as a
         SettingsMS2Deepscore object.
     """
-    if settings.random_seed is not None:
-        np.random.seed(settings.random_seed)
+    # todo why do we need to set a random seed here??
```
Comment (niekdejonge) [inline ms2deepscore/train_new_model/spectrum_pair_selection.py:155]: @florian-huber Why would we need a random seed here? The fingerprint calculations don't have a random component right?

Comment (florian-huber) [reply ms2deepscore/train_new_model/spectrum_pair_selection.py:155]: True. Must be a leftover and can be removed!

File: ms2deepscore/SettingsMS2Deepscore.py [hunk 2]
```diff
@@ -183,3 +177,35 @@ def default(self, o):
...
+        self.evaluator_depth = 3
+        self.evaluator_kernel_size = 20
+        self.random_seed: Optional[int] = None
+
+        # Training settings
+        # todo set actually good settings
+        self.mini_batch_size = 32
+        self.batches_per_iteration = 100
+        self.learning_rate = 0.001
+        self.num_epochs = 200
```
[the remark this answers is not in the document]
Comment (florian-huber) [reply ms2deepscore/SettingsMS2Deepscore.py:200]: Still maybe not the final state, but probably closer...

File: ms2deepscore/models/EmbeddingEvaluatorModel.py [hunk 3]
```diff
@@ -68,46 +70,41 @@ def save(self, filepath):
...
 
-    def train_evaluator(
-            self,
-            data_generator: DataGeneratorEmbeddingEvaluation,
-            mini_batch_size: int,
-            batches_per_iteration: int,
-            learning_rate: float,
-            num_epochs: int,
-            val_generator: DataGeneratorEmbeddingEvaluation = None):
+    def train_evaluator(self,
```
Comment (florian-huber) [inline ms2deepscore/models/EmbeddingEvaluatorModel.py:73]:
```suggestion
    def train(self,
```

Comment (florian-huber) [reply ms2deepscore/models/EmbeddingEvaluatorModel.py:73]: Since it is now part of the model class we don't have to specify what this is training.

Comment (niekdejonge) [reply ms2deepscore/models/EmbeddingEvaluatorModel.py:73]: There is a method of the nn.module which is also called train, which we call in train_evaluator. Which I think would result in us breaking something, but I am not fully sure.

Comment (florian-huber) [reply ms2deepscore/models/EmbeddingEvaluatorModel.py:73]: Oh sorry, that's of course correct. Than nevermind....
